packages/screens/LaunchpadERC20/LaunchpadERC20Sales/LaunchpadERC20CreateSaleForm.tsx

```diff
@@ -0,0 +1,190 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import React, { useEffect } from "react";
+import { useForm } from "react-hook-form";
+import { TouchableOpacity, View } from "react-native";
+
+import { LaunchpadERC20CreateSaleFooter } from "./LaunchpadERC20CreateSaleFooter";
+import { useCreateSaleState, zodCreateSaleForm } from "../hooks/useCreateSale";
```

Comment:
```suggestion
import { useCreateSaleState, zodCreateSaleForm, CreateSaleForm as TCreateSaleForm } from "../hooks/useCreateSale";
```

See `TCreateSaleForm` usage in the suggestions bellow